bresenham_numpy.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from typing import Tuple, Union
+
+
+def bresenham(
+        x0: Union[float, int],
+        y0: Union[float, int],
+        x1: Union[float, int],
+        y1: Union[float, int]
```

Comment:
The algorithm deals with integers, please don't allow `float`. (Or, if this is generalized to `float`, please test that case.)

Reply:
I added tests for floats, but it is tricky.
Can you verify if the tests are satisfactory to you?
What I did is, I took some of the existing tests and moved the starting point by one unit in either X or Y (whichever has shorter delta between start and end point) towards the end point. Then I removed the first point from the results.